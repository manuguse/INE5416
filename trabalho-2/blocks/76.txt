1 6 6 6 14 15 20 20 25 25 25 25
2 7 7 13 14 15 15 21 21 26 25 25
2 8 7 13 15 15 19 22 26 26 29 32
2 8 8 13 15 19 19 22 26 29 29 32
2 8 8 11 11 19 19 22 22 30 29 32
2 2 11 11 16 16 19 22 22 30 30 32
2 9 9 9 9 16 16 23 23 31 30 32
3 3 3 9 9 16 16 23 23 31 32 32
4 10 10 9 17 17 17 17 17 27 33 33
4 4 12 12 18 18 17 17 27 27 28 28
5 4 4 12 18 18 18 18 28 28 28 28
5 5 5 5 5 5 18 24 24 24 24 28