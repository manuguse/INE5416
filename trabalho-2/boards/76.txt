- - - 1 - 5 - 2 3 6 - 5
- - 2 - - - 6 - - - 1 2
- - - - - - - 6 4 - - 7
- - 3 - - - 4 - 1 - - 5
- 1 - - 2 - - 4 - - - -
- 3 4 - 4 6 - - 1 - - 3
1 6 - - - - - - 4 - 1 -
- - 2 - 2 1 - 1 2 - - -
- - 1 - - 5 2 - 6 3 - -
- 4 - 3 - - - 3 - - - 7
6 2 - - - 4 6 - 3 - - 5
4 5 - 2 1 - 2 - 2 3 - 4