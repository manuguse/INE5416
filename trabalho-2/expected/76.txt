1 2 3 1 2 5 1 2 3 6 4 5
7 3 2 3 1 2 6 1 2 3 1 2
6 5 1 2 4 1 5 6 4 2 4 7
5 4 3 1 3 6 4 5 1 3 2 5
4 1 2 3 2 3 2 4 3 4 1 4
2 3 4 1 4 6 1 2 1 3 2 3
1 6 1 7 3 2 5 3 4 2 1 2
3 1 2 5 2 1 3 1 2 1 6 1
5 2 1 4 7 5 2 4 6 3 1 2
1 4 2 3 5 7 1 3 1 2 6 7
6 2 3 1 3 4 6 1 3 1 2 5
4 5 7 2 1 3 2 4 2 3 1 4